src/Microsoft.Extensions.Configuration.UserSecrets/project.json

```diff
@@ -18,13 +18,16 @@
       "configuration",
       "secrets",
       "usersecrets"
-    ]
+    ],
+    "files": {
+      "include": "build/**/*"
+    }
   },
   "dependencies": {
     "Microsoft.Extensions.Configuration.Json": "1.0.0"
   },
   "frameworks": {
     "net451": {},
-    "netstandard1.3": {}
+    "netstandard1.5": {}
```

Comment:
It's not a breaking change. All currently supported netstandard platforms will continue to work.